.github/workflows/test.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Retrieve screenshots
         if: failure()
         run: .circleci/process-artifacts.sh
-        timeout-minutes: 3
+        timeout-minutes: 1
```

Comment:
Ive seen in another plugin where the upload-artifact won't work if there is a ; or : in the filename, you may need to 'zip' the screenshots and give it the file to upload insteadl

Reply:
It doesn't like double quotes, either. I think the scenario names have all been cleaned up now, though.